frappe/utils/install.py

```diff
@@ -241,6 +239,10 @@ def add_country_and_currency(name, country):
 def add_standard_navbar_items():
 	navbar_settings = frappe.get_single("Navbar Settings")
 
+	# don't add settings/help options if they're already present
+	if navbar_settings.settings_dropdown and navbar_settings.help_dropdown:
+		return
+
```

Comment:
added this here because say we have erpnext/any other app installed which extends the navbar items and if we force install frappe then, we'll get a validation error as we check the number of navbar items before and after save.

skipping this function if the items are already present